### PR TITLE
Use request form contact info for the invitation emails

### LINKED ISSUE #938: Recruitment: use request form contact info for the invitation emails
Currently the default email template is:

```
Dear {name},

You have been nominated by the program chair committee of ICML 2021 Workshop AML to serve as {invitee_role}. As a respected researcher in the area, we hope you will accept and help us make ICML 2021 Workshop AML a success.

You are also welcome to submit papers, so please also consider submitting to ICML 2021 Workshop AML.

We will be using OpenReview.net with the intention of have an engaging reviewing process inclusive of the whole community.

To ACCEPT the invitation, please click on the following link:

{accept_url}

To DECLINE the invitation, please click on the following link:

{decline_url}

Please answer within 10 days.

If you accept, please make sure that your OpenReview account is updated and lists all the emails you are using.  Visit http://openreview.net/profile after logging in.

If you have any questions, please contact info@openreview.net.

Cheers!

Program Chairs
```

We need to make to changes:
- Remove info@openreview.net and define a new parameter {contact_email}
- Replace {contact_email} with the value present in `request_form.content.contact_email`
If the PCs override this template, we should force them to have {contact_email} on it.

### PR DESCRIPTION
**Pull request by haroldrubio**
Resolves #938

In `builder.py` and `deployProcess.py`, the request form note is retrieved and searched for the key `contact_email` in the content field. If no contact email is present, defaults to info@openreview.net

### CODE REVIEW

File: openreview/conference/builder.py [hunk 1]
```diff
@@ -1233,7 +1241,7 @@ def recruit_reviewers(self, invitees = [], title = None, message = None, reviewe
 
         If you accept, please make sure that your OpenReview account is updated and lists all the emails you are using.  Visit http://openreview.net/profile after logging in.
 
-        If you have any questions, please contact info@openreview.net.
+        If you have any questions, please contact ''' + contact_email + '''.
```
Comment (melisabok) [inline openreview/conference/builder.py:1244]: Please add `{contact_info}` as part of the template and add the contact_info parameter in `tools.recruit_reviewers`.

File: openreview/conference/builder.py [hunk 2]
```diff
@@ -1212,6 +1212,14 @@ def recruit_reviewers(self, invitees = [], title = None, message = None, reviewe
         invitation = self.invitation_builder.set_reviewer_recruiter_invitation(self, options)
         invitation = self.webfield_builder.set_recruit_page(self.id, invitation, self.get_homepage_options(), options['reviewers_name'])
 
+        contact_email = 'info@openreview.net'
```
Comment (melisabok) [inline openreview/conference/builder.py:1215]: can you add contact_info as a parameter in this function?

The request form recruitment process function should try to get the contact info from the request note and pass it to this function. 

The conference object can be used without a request form note, so it is better to parametrize this value.

File: openreview/conference/builder.py [hunk 3]
```diff
@@ -1212,6 +1212,14 @@ def recruit_reviewers(self, invitees = [], title = None, message = None, reviewe
...
+        try:
+            request_form = self.client.get_note(self.request_form_id)
+            if 'contact_email' in request_form.content.keys():
+                contact_email = request_form.content['contact_email']
```
Comment (melisabok) [inline openreview/conference/builder.py:1219]: here, you can se the get function that doesn't throw a KeyError when the key doesn't exist in the dictionary:

`request_form.content.get('contact_email', None)`

Default value is always None but you can use other values, for example: 'info@openreview.net' but I would prefer to throw an error.

File: openreview/venue_request/process/deployProcess.py [hunk 4]
```diff
@@ -64,6 +64,14 @@ def process(client, note, invitation):
         signatures = ['~Super_User1']
     ))
 
+    contact_email = 'info@openreview.net'
```
Comment (melisabok) [inline openreview/venue_request/process/deployProcess.py:67]: no need to this here.

File: openreview/venue_request/process/deployProcess.py [hunk 5]
```diff
@@ -85,7 +93,7 @@ def process(client, note, invitation):
 
 If you accept, please make sure that your OpenReview account is updated and lists all the emails you are using.  Visit http://openreview.net/profile after logging in.
 
-If you have any questions, please contact info@openreview.net.
+If you have any questions, please contact ''' + contact_email + '''.
```
Comment (melisabok) [inline openreview/venue_request/process/deployProcess.py:96]: use `{contact_info}` instead.

Comment (melisabok) [reply openreview/venue_request/process/deployProcess.py:96]: Once you add the parameter contact_info in conference.recruit_reviewers, you need to pass this parameter here:

https://github.com/openreview/openreview-py/blob/master/openreview/venue_request/process/recruitmentProcess.py#L38

you can verify this is correctly working in the test_neuirps_conference.py, making sure the content of the email has the right contact_info value.

File: openreview/conference/builder.py (hunk 3 above)
Comment (haroldrubio) [reply openreview/conference/builder.py]: As in - throw an error when there is no `contact_info` field in the note?

Comment (melisabok) [reply openreview/conference/builder.py]: Just put the contact_info as a required parameter of recruit_reviewers.

**Comment by haroldrubio** [comment]
I think I have a grasp on the functions now - `deployProcess` creates the invitation, `recruitmentProcess` executes recruiting and calls `conference.recruit_reviewers` which makes subsequent calls to `tools.recruit_reviewer`.

Also, I have found that `tools.recruit_reviewer` is also called in the `journal` module - how should I handle that? I'm not sure if journals have a similar request form that also contains a `contact_info` field.

**Comment by melisabok** [comment]
That's correct, you may need to fix the journal tests or for now we can define a default value and don't make the parameter mandatory. 

```
recruit_reviewers(self, invitees = [], title = None, message = None, reviewers_name = 'Reviewers', remind = False, invitee_names = [], retry_declined=False, contact_info='info@openreview.net')

```

But make sure we pass a value in the recruitmentProcess.

File: openreview/venue_request/process/recruitmentProcess.py [hunk 6]
```diff
@@ -34,13 +34,22 @@ def process(client, note, invitation):
         'area chair': 'Area_Chairs',
         'senior area chair': 'Senior_Area_Chairs'
     }
+
+    # Fetch contact info
+    if conference.request_form_id:
```
Comment (melisabok) [inline openreview/venue_request/process/recruitmentProcess.py:39]: in the line 4 of this process function you already have `request_form` so no need to get the note again.

**Comment by melisabok** [comment]
it looks good!!!! I requested a small change, @celestemartinez do you want to take a look?

**Review by celestemartinez — APPROVED** [review]

**Pull request merged**